docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -180,6 +180,8 @@ primary shard that was previously allocated.
 ----
 // NOTCONSOLE
 
+The shard in the example above errors `no_valid_shard_copy` due to `NODE_LEFT`. The first step to recover is to make sure all nodes are in the cluster. Then, if the error continues after a <<cluster-reroute,cluster reroute>>, the data will need to be <<snapshots-restore-snapshot,restored from snapshot>>.
```

Comment:
I think this info is valuable, but I don't think that it's the appropriate location for the recovery steps. We have a [page that talks about recovering lost nodes](https://www.elastic.co/guide/en/elasticsearch/reference/current/red-yellow-cluster-status.html#fix-cluster-status-recover-nodes) - we might consider linking to that doc, and enhancing it as needed. This way, all of the info is centralized. Let me know what you think.

Reply:
👋🏽 I'm on board to either page as long as we can write an actionable commentary for this response example. (I helped Annie write her first doc PR!) The Support back story is that users search for the explicit error and `no_valid_shard_copy` only shows on this page. 

We could just link or also add the error's literal string on the [recovery data for a lost primary shard](https://www.elastic.co/guide/en/elasticsearch/reference/current/red-yellow-cluster-status.html#fix-cluster-status-restore). If we cross-link/pollinate, could y'all also kindly consider reformatting the destination so these don't look like sequential steps (which'd destroy your data right after recovering it?):
<img width="445" alt="image" src="https://github.com/elastic/elasticsearch/assets/26751266/f50bec37-f826-47ac-82a5-67c42ab8f84a">